vpype/hpgl_devices.toml

```diff
@@ -40,3 +40,38 @@ y_range = [0, 10365]
 y_axis_up = true
 origin_location = ["15mm", "287mm"]
 set_ps = 0
+
+[device.calcomp_designmate]
+name = "designmate"
+plotter_unit_length = "0.02488mm"
+pen_count = 8
+
+[[device.calcomp_designmate.paper]]
+name = "a4"     # Will always be loaded in Landscape oriebtaion in the Designmate
+aka_names = ["A4"]
+paper_size = ["297mm", "210mm"]
+x_range = [0, 11040]
+y_range = [0, 7721]
+y_axis_up = true
+origin_location = ["12mm", "198mm"]
+set_ps = 0
```

Comment:
Does it really make sense for a `PS0;` to be emitted? I feel it's not the case. I'd remove this if so (so no PS is emitted).

Reply:
done